roles/build_openstack_packages/tasks/install_dlrn.yml

```diff
@@ -72,10 +72,19 @@
   changed_when: result.stdout or result.stderr
 
 - name: Create dlrn-venv
+  vars:
+    _upstream_command: "/usr/bin/python3 -m venv"
+    _downstream_command: "{{ _upstream_command }} --system-site-packages"
```

Comment:
it seems there's a recursion issue - maybe you shouldn't define that _downstream_command and, in the block below, just append that --system-site-packages to the upstream_command?